lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -526,13 +526,13 @@ public function notLike($x, $y)
      * Creates a CONCAT() function expression with the given arguments.
      *
      * @param mixed $x First argument to be used in CONCAT() function.
-     * @param mixed $y Second argument to be used in CONCAT() function.
+     * @param mixed $y,... Other arguments to be used in CONCAT() function.
      *
      * @return Expr\Func
      */
     public function concat($x, $y)
```

Comment:
Sure, I kept the minimum of 2 not to introduce BC. Not supporting 2+ arguments is a bug, since CONCAT DQL expression is now valid syntax, but supporting a single parameters is halfway a bugfix and a feature.

I'll do some test and commit the new change so that the maintainer can review.